v2/api-validator/src/server/controllers/withdrawal-controller.ts

```diff
@@ -88,6 +90,45 @@ export class WithdrawalController {
       loadCapabilitiesJson(`withdrawals-${this.accountId}.json`) ??
       WithdrawalController.generateWithdrawalCapabilities();
 
+    const collateralWithdrawalCapabilities: WithdrawalCapability[] = [
```

Comment:
This should be part of `WithdrawalController.generateWithdrawalCapabilities()`. I also recommend to randomize it.

Also add it please to `all-capabilities` capability preset. I recommend also generating a separate preset for off-exchange.